app/controllers/webhooks/stripe/invoices_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Webhooks::Stripe::InvoicesController < Webhooks::BaseController
+  skip_before_action :verify_authenticity_token, only: [:create, :charge_event]
```

Comment:
Here you can probably set

```ruby
protect_from_forgery with: :null_session
```

I'd actually put in the base webhooks controller. We shouldn't be verifying the authenticity token at all in webhooks.